homeassistant/components/niu/translations/it.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Remove this file. Translations for other languages are automatically updated through Lokalise: https://developers.home-assistant.io/docs/internationalization/core